### ISSUE TITLE
Doesn't work with the latest MCreator

### ISSUE THREAD

**Issue by OscarHdez103** [Goldorion/MCreatorFabricGenerator#498]
### Issue description

According to the readme, "Make sure you use MCreator 2023.3 or a newer version" it should work with newer versions but it breaks because of the version check. I managed to make it work just by modifing the plugin.json file to include the latest (2023.4) MCreator version and everything seems to work but that change is needed for it to load.

### How to reproduce this issue?

Update MCreator and run it with this plugin installed it should get a error message when loading

### Operating system

Windows

### Details

I am using a window laptop

### Logs

Make sure you use MCreator 2023.3 or a newer version

### Issue tracker rule checks (please read carefully)

- [X] I have checked that my problem <a href='https://github.com/MCreator/MCreator/issues?q=is%3Aissue' target='_blank'>is not already reported</a>
- [X] I have checked that my problem is not covered on <a href='https://mcreator.net/support/knowledgebase' target='_blank'>Knowledge Base</a> or on <a href='https://mcreator.net/wiki' target='_blank'>MCreator's Wiki</a>
- [X] I have checked that my written content does not violate the <a href='https://mcreator.net/wiki/general-publishing-guidelines' target='_blank'>publishing guidelines</a>

**Comment by Goldorion** [comment]
If you had to change the supported MCR versions, this means you shouldn't have changed the supported MCR versions. If I block the plugin to a specific version, there's a reason and the reason is it simply doesn't work because I never changed something to make it work with the latest version.

**Comment by smmmadden** [comment]
I think he was only pointing out that the plugin will work with just changing the MCR version.  Obviously, no one should do this but in this case it might be all that is needed unless you know of any scenario that it will not.  I read this more as an FYI as help rather than this is how to fix it.  :-)

**Comment by Goldorion** [comment]
> I think he was only pointing out that the plugin will work with just changing the MCR version.

It won't. it's just impossible. I worked on the 2023.4 version (as I'm one of the main contributors) and there's no way it can work correctly without changing something. Way too many things changed, and way too many things got added and removed. We often said during its development it would be the biggest update and we often say it with big updates, but it wasn't a lie. It really is the biggest update MCreator has received since its creation. An update never got so many giant features. They might not look like something big, but as someone who knows what's happening inside the code, there are far more big features than users think. (Just the split of ranged items into the projectile mod element and the new ranged features for items will break  workspaces).
